main/src/cgeo/geocaching/connector/ConnectorFactory.java

```diff
@@ -168,6 +168,17 @@ private ConnectorFactory() {
         return liveConns.toArray(new ILogin[liveConns.size()]);
     }
 
+    @NonNull
+    public static IConnector[] getActiveConnectors() {
```

Comment:
Why not return directly the `List<IConnector>`?

Reply:
No preference from my side, just wanted to be consistent with existing `getActiveLiveConnectors` which returns `ILogin[]`. SHould I change to `List<IConnector>`?

Reply:
The simplest the better, so probably.

Reply:
Done